src/lib/utilities/route-for-api.ts

```diff
@@ -11,14 +13,15 @@ const replaceNamespaceInApiUrl = (
   return '';
 };
 
-const base = async (namespace?: string): Promise<string> => {
+const base = (namespace?: string): string => {
   let baseUrl = '';
+  const webUrl: string | undefined = get(page).data?.webUrl;
 
-  if (globalThis?.GetNamespaces && namespace) {
-    const namespaces = await globalThis.GetNamespaces();
-    const configNamespace = namespaces?.find((n) => n.namespace === namespace);
+  // Use GetNamespace webUrl or fallback env var
+  const apiUrl = webUrl || globalThis?.AppConfig?.apiUrl;
+  if (apiUrl && namespace) {
     baseUrl =
-      configNamespace?.webUri ??
+      webUrl ||
       replaceNamespaceInApiUrl(globalThis?.AppConfig?.apiUrl, namespace);
```

Comment:
Some kind of logging here would probably be a good idea so we can indicate when the fallback api is being used.

Reply:
👍 Made this logic easier to grok and added a console.warn if using the fallback